ego/cmd/integration-test/main.go

```diff
@@ -95,3 +97,7 @@ func testEnvVars(assert *assert.Assertions, require *require.Assertions) {
 	// Check if other env vars were not taken over by using a common one to check against (here: LANG)
 	assert.Empty(os.Getenv("LANG"))
 }
+
+func testCpuid(assert *assert.Assertions, require *require.Assertions) {
+	assert.True(cpuid.CPU.Has(cpuid.CMOV))
+}
```

Comment:
```suggestion
func testCpuid(assert *assert.Assertions) {
	assert.True(cpuid.CPU.Has(cpuid.CMOV))
}
```

Unless we want `require` for style reasons, it can be removed here

Reply:
Yes, I wanted to have consistent function signatures. But I only have a slight preference for this way.